platform/kubernetes/files.go

```diff
@@ -22,15 +22,20 @@ func FindDefinedServices(path string) (map[flux.ServiceID][]string, error) {
 	}
 
 	var files []string
-	filepath.Walk(path, func(target string, info os.FileInfo, err error) error {
+	if err = filepath.Walk(path, func(target string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
 		if info.IsDir() {
 			return nil
 		}
 		if ext := filepath.Ext(target); ext == ".yaml" || ext == ".yml" {
 			files = append(files, target)
 		}
 		return nil
-	})
+	}); err != nil {
+		return nil, err
```

Comment:
Aye, it should percolate out of  `SomeServices` and get logged by `loggingPlatform`